lib/paypal_express/api.rb

```diff
@@ -266,7 +266,7 @@ def process_notification(notification, properties, context)
       def to_express_checkout_url(response, kb_tenant_id, options = {})
         payment_processor_account_id = options[:payment_processor_account_id] || :default
         gateway                      = lookup_gateway(payment_processor_account_id, kb_tenant_id)
-        review                       = options.has_key?(:review) ? options[:review] : false
+        review                       = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(options, :review)
         gateway.redirect_url_for(response.token, :review => review)
```

Comment:
Yes it's handled [here](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/paypal_express_common.rb#L24)